### ISSUE TITLE
Cleanup autosklearn warnings

### ISSUE THREAD

**Issue by bjschoenfeld** [byu-dml/d3m-dynamic-neural-architecture#60]
```
.../d3m-dynamic-neural-architecture/env/lib/python3.6/site-packages/autosklearn/evaluation/train_evaluator.py:197: RuntimeWarning: Mean of empty slice
  Y_train_pred = np.nanmean(Y_train_pred_full, axis=0)
[WARNING] [2019-07-03 10:45:47,451:AutoMLSMBO(0)::f78924aaf4621a84a09b4eaff3dd4995] Could not find meta-data directory .../d3m-dynamic-neural-architecture/env/lib/python3.6/site-packages/autosklearn/metalearning/files/r2_regression_dense
[WARNING] [2019-07-03 10:45:47,459:EnsembleBuilder(0):f78924aaf4621a84a09b4eaff3dd4995] No models better than random - using Dummy Score!
[WARNING] [2019-07-03 10:45:47,462:EnsembleBuilder(0):f78924aaf4621a84a09b4eaff3dd4995] No models better than random - using Dummy Score!
.../d3m-dynamic-neural-architecture/env/lib/python3.6/site-packages/autosklearn/evaluation/train_evaluator.py:197: RuntimeWarning: Mean of empty slice
  Y_train_pred = np.nanmean(Y_train_pred_full, axis=0)
.../d3m-dynamic-neural-architecture/env/lib/python3.6/site-packages/autosklearn/evaluation/train_evaluator.py:197: RuntimeWarning: Mean of empty slice
  Y_train_pred = np.nanmean(Y_train_pred_full, axis=0)
```

**Comment by orionw** [comment]
This goes away if you give autosklearn more time.  Without it, sometimes it has no predictions and thus tries to compute accuracy on an empty predictions array.  

However, we can just remove all warnings that come from autosklearn.  I just wanted you to know what the issue was.

**Comment by orionw** [comment]
We decided to leave the warning there because we didn't want to hide all warnings from these function calls.
